app/models/get_an_identity/external/user.rb

```diff
@@ -0,0 +1,56 @@
+module GetAnIdentity
+  module External
+    class User
+      class InvalidTokenError < StandardError; end
+      class NotFoundError < StandardError; end
+
+      def self.find(id)
+        new(id:)
+      end
+
+      attr_reader :id,
+                  :uid,
+                  :email,
+                  :full_name,
+                  :date_of_birth,
+                  :trn,
+                  :mobile_number,
+                  :merged_user_ids,
+                  :raw
+
+      def initialize(id:)
+        @id = id
+        self.load
+      end
+
+    private
+
+      def request_user
```

Comment:
Im not sure if we will need many more of these services going forward but i think if that happened we should re factor some of this work to have a base class that would contain a lot of this type of code?

I think can let it slide for this PR but as you mentioned this would be more than just grabbing data in the future hence keeping as a model. 

so refactor things like the token fetching and maybe just having a structure to pass into the load class etc